org.sugarlabs.Memorize.json

```diff
@@ -39,8 +39,8 @@
                     "x-checker-data": {
                         "type": "git",
                         "tag-pattern": "^([\\d.]+)$"
-                   }
-               }
+                    }
+                }
```

Comment:
Are these fixes or typos?

Reply:
These are formatting fixes.

Reply:
For future reference, it's better to keep these fixes in a separate commit. But let's go with this now.